.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "checksum_reader"]
```

Comment:
Not in pypi at this point. Would be great to get that up there actually this whole submodule thing is annoying. @sampierson I can look at that with you tomorrow if you have time.